soh/soh/Enhancements/debugger/actorViewer.cpp

```diff
@@ -918,122 +887,104 @@ void ActorViewerWindow::DrawElement() {
 
         PushStyleHeader(THEME_COLOR);
         if (ImGui::TreeNode("Selected Actor")) {
-            DrawGroupWithBorder(
-                [&]() {
-                    ImGui::Text("Name: %s", ActorDB::Instance->RetrieveEntry(display->id).name.c_str());
-                    ImGui::Text("Description: %s", GetActorDescription(display->id).c_str());
-                    ImGui::Text("Category: %s", acMapping[display->category]);
-                    ImGui::Text("ID: %d", display->id);
-                    ImGui::Text("Parameters: %d", display->params);
-                },
-                "Selected Actor");
-            ImGui::SameLine();
-            ImGui::PushItemWidth(ImGui::GetFontSize() * 6);
-
-            DrawGroupWithBorder(
-                [&]() {
-                    ImGui::PushItemWidth(ImGui::GetFontSize() * 6);
-                    PushStyleInput(THEME_COLOR);
-                    ImGui::Text("Actor Position");
-                    ImGui::InputScalar("X##CurPos", ImGuiDataType_Float, &display->world.pos.x);
-                    ImGui::InputScalar("Y##CurPos", ImGuiDataType_Float, &display->world.pos.y);
-                    ImGui::InputScalar("Z##CurPos", ImGuiDataType_Float, &display->world.pos.z);
-                    ImGui::PopItemWidth();
-                    PopStyleInput();
-                },
-                "Actor Position");
-            ImGui::SameLine();
-            DrawGroupWithBorder(
-                [&]() {
+            if (display != nullptr) {
```

Comment:
I'm now hiding the actor information until one is actually selected. I always disliked the "dummy" actor solution, because it threw me off that it wasn't an actual actor in the world, and that it was somehow "Link" in the "Switch" category...